helm-charts/hbase-chart/templates/meta/_zkscript.tpl

```diff
@@ -17,8 +18,63 @@ function shutdown() {
   echo "Stopping Zookeeper"
   $HBASE_HOME/bin/hbase-daemon.sh stop zookeeper
   if [ $exit_status != 0 ]; then
-	  echo "Leader stopped, sleeping for 60 seconds"
-	  sleep 60
+     if [ ! -f "$HBASE_CONF_DIR/hbase-site.xml" ]; then
+        echo "$HBASE_CONF_DIR/hbase-site.xml does not exists"
+        sleep 120
+        exit 1
```

Comment:
can you verify once this happens? make sure exit 1 vs exit 0 is not behaving differently

Reply:
Looks to be working fine